### PR TITLE
WRP-7819 : Update README for the detailed information

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update README in babel-preset-enact with the detailed information

References
- https://github.com/enactjs/eslint-config-enact/blob/master/README.md
- https://github.com/enactjs/eslint-config-enact-proxy/blob/master/README.md
- https://github.com/facebook/create-react-app/blob/main/packages/babel-preset-react-app/README.md

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Added `Usage in Enact projects created by Enact CLI`
- Added `Usage Outside of Enact projects`
- Added `Copyright and License Information` 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-7819

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+### Checklist
```
Comment (MikyungKim) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Would you please make another PR for this targeting `main`? Since it does not apply when it is in `develop` branch.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Documented `Usage` and `Copylight and License Information` in README.md.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]: As I was saying, we don't add a change log for documentation.

File: README.md [hunk 3]
```diff
@@ -2,9 +2,39 @@
...
 ```sh
 $ npm i --save babel-preset-enact
 ```
 
+## Copyright and License Information
+
+Unless otherwise specified, all content, including all source code files and
+documentation files in this repository are:
+
+Copyright (c) 2020-2022 LG Electronics
```
Comment (MikyungKim) [inline README.md:26]: Why this period is 2020-2022?

File: README.md [hunk 4]
```diff
@@ -2,9 +2,39 @@
 
 > This package includes the Babel preset used by Enact.
 
+## Usage
+
+Reference this module from your `babel.config.js`.
+
+```js
+module.exports = require('babel-preset-enact');
```
Comment (MikyungKim) [inline README.md:10]: I think CRA describes well with this. Would you please add `Usage in Enact projects created by Enact CLI` and `Usage Outside of Enact projects`?
Also, it would be better to present usage with `preset` instead of this line.

File: README.md [hunk 5]
```diff
@@ -2,9 +2,39 @@
...
+```
+
+For more information (including `babel.config.js` setup), please see the [babel docs](https://babeljs.io/docs/config-files).
```
Comment (MikyungKim) [inline README.md:13]: Try not to use ( ) in this phase. Would you rewrite this, please?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (MikyungKim) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I'll take care of this.

Comment (hong6316) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: OK

File: CHANGELOG.md (hunk 2 above)
Comment (hong6316) [reply CHANGELOG.md:3]: OK

File: README.md (hunk 3 above)
Comment (hong6316) [reply README.md:26]: I will fix it.
I think we need to add `update-copyright-years.yml` too.

File: README.md (hunk 4 above)
Comment (hong6316) [reply README.md:10]: OK

File: README.md (hunk 5 above)
Comment (hong6316) [reply README.md:13]: OK

File: README.md [hunk 6]
```diff
@@ -2,9 +2,49 @@
...
+First, [install Babel](https://babeljs.io/docs/setup/).
+
+Then install babel-preset-enact.
 
 ```sh
-$ npm i --save babel-preset-enact
+npm install babel-preset-enact --save-dev
+```
+
+Then create a file named `.babelrc` with following contents in the root folder of your project:
```
Comment (MikyungKim) [inline README.md:21]: Should `.babelrc` be `.babelrc.json` or not?

File: README.md [hunk 7]
```diff
@@ -2,9 +2,49 @@
...
+
+```json
+{
+  "presets": ["enact"]
```
Comment (MikyungKim) [inline README.md:25]: Should this be
```suggestion
  "presets": ["babel-preset-enact"]
```
?

File: README.md [hunk 8]
```diff
@@ -2,9 +2,49 @@
...
+}
 ```
 
+For more information including `.babelrc` setup, please see the [babel docs](https://babeljs.io/docs/config-files)
```
Comment (MikyungKim) [inline README.md:29]:
```suggestion
For more information including `.babelrc` setup, please see the [Babel docs](https://babeljs.io/docs/config-files)
```

File: README.md [hunk 9]
```diff
@@ -2,9 +2,49 @@
...
+
+## Copyright and License Information
+
+Unless otherwise specified, all content, including all source code files and
+documentation files in this repository are:
+
+Copyright (c) 2020-2023 LG Electronics
```
Comment (MikyungKim) [inline README.md:36]: 🤔 Should this year be 2023? Why it's from 2020?

File: README.md (hunk 6 above)
Comment (hong6316) [reply README.md:21]: Either is fine, Both are the same.
And CRA also described `.babelrc`.

File: README.md (hunk 9 above)
Comment (hong6316) [reply README.md:36]: You're right.
I will change this.

File: README.md [hunk 10]
```diff
@@ -2,9 +2,49 @@
...
+
+Then create a file named `.babelrc` with following contents in the root folder of your project:
+
+```json
+{
+  "presets": ["babel-preset-enact"]
+}
 ```
 
+For more information including `.babelrc` setup, please see the [Babel docs](https://babeljs.io/docs/config-files)
```
Comment (MikyungKim) [inline README.md:29]:
```suggestion
For more information including `.babelrc` setup, please see the [Babel docs](https://babeljs.io/docs/config-files).
```

File: README.md (hunk 6 above)
Comment (MikyungKim) [inline README.md:21]:
```suggestion
Then create a file named `.babelrc` with the following contents in the root folder of your project:
```

**Review by MikyungKim — APPROVED** [review]
LGTM
